### PR TITLE
fix: use the correct resvg-js parameters to avoid PNG jaggies

### PR DESCRIPTION
**Pull request by yisibl**
In resvg-js, only need to enter the final desired width or height via the `fitTo` parameter.

### CODE REVIEW

File: scripts/benchmark/resvg.js [hunk 1]
```diff
@@ -6,13 +6,9 @@ export default svg => (
 			loadSystemFonts: false,
 		},
 		logLevel: 'off',
-		imageRendering: 1,
-		textRendering: 0,
-		shapeRendering: 0,
```
Comment (privatenumber) [inline scripts/benchmark/resvg.js:11]: I was hoping these values would help speed up the process.

File: scripts/benchmark/resvg.js [hunk 2]
```diff
@@ -6,13 +6,9 @@ export default svg => (
...
 		fitTo: {
 			mode: 'width',
 			value: 800,
 		},
-		dpi: 2400,
```
Comment (privatenumber) [inline scripts/benchmark/resvg.js:16]: How can I make resvg-js set the output DPI to 2400?

Seems adding this back doesn't really make a difference. The resulting DPI is still 72.

#### resvg-js
Screen Shot 2022-08-11 at 1 09 55 AM

#### sharp
Screen Shot 2022-08-11 at 1 10 16 AM

File: scripts/benchmark/resvg.js (hunk 1 above)
Comment (yisibl) [reply scripts/benchmark/resvg.js:11]: This will change the rendering result of the PNG

File: scripts/benchmark/resvg.js (hunk 2 above)
Comment (yisibl) [reply scripts/benchmark/resvg.js:16]: See: https://github.com/RazrFalcon/resvg/issues/526#issuecomment-1190433890

Comment (yisibl) [reply scripts/benchmark/resvg.js:16]: And https://github.com/RazrFalcon/resvg/issues/451#issuecomment-914478375

**Pull request closed without merging**